From: Andreas Stieger <Andreas.Stieger@gmx.de>
Subject: [PATCH] skip float precision related test failures on 32 bit
Date: Fri, 01 Jul 2022 21:12:29 +0000
Upstream: no

Introduced by: https://sqlite.org/src/info/c48a735bd4a1dbd5
Upstream thread: https://sqlite.org/forum/forumpost/dd4767885a

[  642s] ! func7-pg-181 expected: [2.000000000000000000000000000000]
[  642s] ! func7-pg-181 got:      [1.999999999999999000000000000000]

Index: sqlite-src-3390000/test/func7.test
===================================================================
--- sqlite-src-3390000.orig/test/func7.test
+++ sqlite-src-3390000/test/func7.test
@@ -60,12 +60,6 @@ do_execsql_test func7-pg-170 {
 do_execsql_test func7-pg-180 {
   SELECT log10(1000.0)
 } {3.0}
-do_execsql_test func7-pg-181 {
-  SELECT format('%.30f', log10(100.0) );
-} {2.000000000000000000000000000000}
-do_execsql_test func7-pg-182 {
-  SELECT format('%.30f', ln(exp(2.0)) );
-} {2.000000000000000000000000000000}
 do_execsql_test func7-pg-190 {
   SELECT log(2.0, 64.0)
 } {6.0}
